cmd/log-parser/README.md

```diff
@@ -0,0 +1,94 @@
+# `kata-log-parser`
+
+* [Logfile requirements](#logfile-requirements)
+* [Component logfiles](#component-logfiles)
+* [Usage](#usage)
+
+`kata-log-parser` is a tool that combines logfiles generated by the various
+system components, sorts them by timestamp and re-displays the log entries,
```

Comment:
Lines 8-9 suggested rewrite: system components, sorts them by timestamp, and re-displays the log entries. A time delta is added to shows how much time has elapsed between each log entry.